### ISSUE TITLE
Enhance vaccination certificate acceptance for outside of Europe

### ISSUE THREAD

**Issue by DRE-TSI** [corona-warn-app/cwa-wishlist#574]
At the moment (July 2021) the vaccination certificate is convenient for traveling in the region of Europe (e.g. within the countries accepting the European vaccination certificate).

But the global travel behaviour also has the need to certify the vaccination status, when traveling as European citizen to e.g. Asia, Africa or America. Or a outer european citrizen, traveling through Europe probably wants to certify his/her vacc status to the european authorities using an accepted European smartphone app, e.g. CWA.

So this wish is to enhance the possibility to scan vaccination certificates from outside of Europe for EU-acceptance. 

(see: https://www.japantimes.co.jp/news/2021/05/24/national/ana-iata-coronavirus-app-trial/)

**Comment by idoodler** [comment]
There is such a ~~standard~~ **thing** called SMART Health Cards: https://smarthealth.cards

Starting in iOS 15 every iOS device can just scan the QR Code and validate the status. Android also supports this, altho being it with special Apps i think.

**Comment by jucktnich** [comment]
This standard can do less in comparison to the EU certificates? Who made this a standard? The W3C? (I couldn't find something on their site)

**Comment by idoodler** [comment]
True, its not a standard, but an already existing solution that works on iOS and Android.

**Comment by jucktnich** [comment]
Yeah, but it shouldn't be a problem for apple/Google to integrate the eu certificates. Maybe the eu can talk to the W3C to make this a standard. Cause I think there should be one, and the EU Certificate is the most spread one and as much as I understand the best standard.

**Comment by idoodler** [comment]
Is there some technical documentation for the EU Certificates? I don't know where exactly the difference is, i just know that they currently are incompatible.

**Comment by MikeMcC399** [comment]
@idoodler 
> Is there some technical documentation for the EU Certificates?

https://www.coronawarn.app/en/faq/#eu_dcc => 
https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en => 
https://ec.europa.eu/health/ehealth/covid-19_en including "Trust framework and detailed technical specifications"

**Comment by ndegendogo** [comment]
as far as I understand there is also standardizing activity at the WHO, only the European health certificate activities didn't want to wait for them.
But maybe W3C should align with WHO before creating yet another standard?
